docs/instance_groups.md

```diff
@@ -242,3 +242,26 @@ spec:
               - http://local-mirror.mydomain
               - http://archive.ubuntu.com
 ```
+
+## Add Tags on AWS autoscalling group and instance
```

Comment:
A nitpick that you can do in a follow-up PR or someone else can edit.  These labels should work in GCE as well now.

Reply:
When I check the godocs (https://godoc.org/k8s.io/kops/pkg/apis/kops#InstanceGroupSpec), there is just mentin of AWS

```// CloudLabels indicates the labels for instances in this group, at the AWS level
    CloudLabels map[string]string `json:"cloudLabels,omitempty"```

Reply:
I am pretty certain the godoc are out of date :)  We are good for now.